source/manage/cloud-team-and-channel.rst

```diff
@@ -23,6 +21,22 @@ Team Profile
 
 The name and description of the team.
 
+Within a Team Profile, System Admins can archive or unarchive the team. Archiving a team makes its contents inaccessible for all users. All related information is archived, including posts from the database. Before archiving a team, we recommend that you perform a database backup.
```

Comment:
Team Profile is not clear to me.  Should we adjust to Team Settings, which is what it is called in the UI?

Reply:
This is the UI that the doc updates are based on:
![Screen Shot 2021-08-05 at 4 43 37 PM](https://user-images.githubusercontent.com/74422101/128418595-9c1a8863-aef9-4e7c-b662-dada0e525e40.png)

Once the System Admin selects a team, the first section of team settings is **Team Profile**, followed by **Team Management**.

Reply:
^ @wiersgallak

Reply:
@cwarnermm My miss on the larger context of where this was in the docs.  Team Profile makes perfect sense when scan out to see the whole picture.  Thanks!